### PR TITLE
V13 cookbook - new section: explicit decoding / encoding of values.

### PR DESCRIPTION
**Pull request by andreibancioiu**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [x] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md [hunk 1]
```diff
@@ -850,6 +850,68 @@ const [sum] = controller.parseQueryResponse(response);
...
+When needed, you can use the [`BinaryCodec`](https://multiversx.github.io/mx-sdk-js-core/v13/classes/BinaryCodec.html) to [decode and encode values](/developers/data/serialization-overview/) manually.
+
+### Decoding a custom type
+
+Example of decoding a custom type (a structure) called `DepositEvent` from binary data:
+
+```
+import { BinaryCodec } from "@multiversx/sdk-core";
+
+const depositCustomType = abi.getCustomType("DepositEvent");
```
Comment (popenta) [inline docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md:864]: maybe also instantiate the `abi` object here?

File: docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md [hunk 2]
```diff
@@ -850,6 +850,68 @@ const [sum] = controller.parseQueryResponse(response);
...
+
+[decoded] = codec.decodeNested(data, rewardStructType);
+decodedValue = decoded.valueOf();
+console.log(JSON.stringify(decodedValue, null, 4));
+```
+
+Example of decoding a custom type (an enum) called `Action` (of [**multisig**](https://github.com/multiversx/mx-contracts-rs/tree/main/contracts/multisig) contract) from binary data:
+
+```
+const actionStructType = abiMultisig.getEnum("Action");
```
Comment (popenta) [inline docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md:887]: also instantiate `abiMultisig` here as well. can be instantiated [here](https://github.com/multiversx/mx-docs/blob/v13-codec/docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md?plain=1#L779), as well.

File: docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md (hunk 1 above)
Comment (andreibancioiu) [reply docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md]: Fixed.

File: docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md (hunk 2 above)
Comment (andreibancioiu) [reply docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md:903]: Fixed.

**Review by popenta — APPROVED** [review]

**Review by danielailie — APPROVED** [review]

**Pull request merged**